astropy/utils/iers/tests/test_iers.py

```diff
@@ -109,6 +109,28 @@ def test_open_network_url(self):
         iers.IERS_A.close()
 
 
+def test_IERS_B_old_style_excerpt():
+    """Check that the instructions given in `IERS_B.read` actually work."""
+    # If this test is changed, be sure to also adjust the instructions.
+    #
+    # TODO: this test and the note can probably be removed after
+    # enough time has passed that old-style IERS_B files are simply
+    # not around any more, say in 2025.  If so, also remove the excerpt
```

Comment:
I think we should open this TODO note as a proper GitHub issue and mention that issue here. Otherwise, we might forget. What do you think?